roles/ciao-controller/README.md

```diff
@@ -1,40 +1,60 @@
-ciao-controller
-=========
+# ciao-controller
 Ansible role to install the controller node for a CIAO cluster
 
 This role configures the following components
 
 * ciao scheduler
 * ciao controller
 
-Requirements
-------------
-None
-
-Role Variables
---------------
-* `ciao_csr_password`
-* `ciao_ssh_public_key`
-* `ciao_admin_email`
-* `ciao_cert_organization`
-
-Dependencies
-------------
-* `ciao-common`
-
-Example Playbook
-----------------
-
+## Requirements
+- A previously setup of the OpenStack Identity service.
+  You can setup one with `os-keystone` role
```

Comment:
[same comment](https://github.com/clearlinux/clear-config-management/pull/20/files#r70143074) applies here (point to URL of the roles)